fake-snippets-api/lib/db/schema.ts

```diff
@@ -100,6 +100,39 @@ export const orderFileSchema = z.object({
 })
 export type OrderFile = z.infer<typeof orderFileSchema>
 
+export const orderQuoteComponentSchema = z.object({
+  component_id: z.string(),
+  name: z.string(),
```

Comment:
You can’t have both a name and a quantity, if the name is the same there is at most quantity 1

btw it might be easier to just use the bom row format, which has each name on a separate line. I dont know where that is spec’d though- maybe in corcuit-json-to-bom

```suggestion
```